darity.go

```diff
@@ -228,6 +237,22 @@ func (v *VM) AddMemorySlot(n uint64, flags MemorySlotFlag) error {
 	return nil
 }
 
+// AddVCPU adds n VCPUs to a virtual machine.
+func (v *VM) AddVCPU(n uint64) error {
```

Comment:
This could probably just be a regular `int`, but that would mean also having to verify that values of less than 0 are invalid.  Maybe just `uint` or `uint8` instead?  `uint64` is overkill for the range 0-9.